.pre-commit-config.yaml

```diff
@@ -1,12 +1,10 @@
 repos:
-- repo: https://github.com/psf/black
-  rev: 24.4.2
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.5.6
   hooks:
-  - id: black
-    args: ['--line-length', '110']
-- repo: https://github.com/pycqa/flake8
-  rev: 7.1.0
-  hooks:
-  - id: flake8
-    args: ['--max-line-length', '110']
-    additional_dependencies: [flake8-bugbear]
+    # Run the linter.
+    - id: ruff
+      args: [ --fix ]
+    # Run the formatter.
+    - id: ruff-format
```

Comment:
Remove chatgpt comments 😂

Reply:
They are not ChatGPT comments, it's the "official" file from upstream, check https://github.com/astral-sh/ruff-pre-commit :)

Reply:
Shit! You win, this time =)